src/Checkbox/checkbox.style.js

```diff
@@ -0,0 +1,59 @@
+import styled from "styled-components";
+
+const CheckboxStyle = styled.div`
+  display: flex;
+  align-items: center;
+
+  label {
+    padding: 0;
+    font-size: 24px;
+    font-weight: normal;
+    line-height: 1.38;
+    letter-spacing: normal;
+    color: #c9c9c9;
+    margin-right: 16px;
+    cursor: pointer;
+  }
+
+  input[type="checkbox"] {
+    display: none;
+  }
+
+  input[type="checkbox"] + label:after {
+    font-family: "vivy-icons";
+    content: "\\e80f";
```

Comment:
isnt there a better way to do that so that we dont have to copy paster these "\\e80f" around?
mb import it in the file and use it as variable?